example/src/main/kotlin/com/example/afterpay/receipt/CashReceiptFragment.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.afterpay.receipt
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.activity.addCallback
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.example.afterpay.NavGraph
+import com.example.afterpay.R
+import com.example.afterpay.data.CashData
+
+class CashReceiptFragment : Fragment() {
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View? = inflater.inflate(R.layout.fragment_cash_receipt, container, false)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        requireActivity().onBackPressedDispatcher.addCallback(this) {
+            findNavController().navigate(NavGraph.action.back_to_shopping)
+        }
+
+        val responseData = requireNotNull(arguments?.get(NavGraph.args.cash_response_data) as CashData)
+        val (cashTag, amount, grantId) = responseData
+
+        context?.let { ctx ->
```

Comment:
This check for safe `context` is not needed. We're inflating things from the `view` that comes as a parameter, so we know for sure that `view` is not null, plus we can just simply call `getString()` no need to use `context.getString()` since this is a Fragment.

Reply:
fixed in [96a2406](https://github.com/afterpay/sdk-android/pull/184/commits/96a2406e3023a8f48582f6bb53572c483983ebe2)